opentreemap/treemap/units.py

```diff
@@ -98,6 +98,9 @@ def convert_to_database_units(self):
 }
 _unit_conversions["ft"] = {u: v * 12 for (u, v)
                            in _unit_conversions["in"].iteritems()}
+_unit_conversions["sq_ft"] = {
+    u: v / _unit_conversions["sq_m"]["sq_ft"]
+    for u, v in _unit_conversions["sq_m"].iteritems()}
```

Comment:
If we switch to storing `drainage_area`, I think we can remove `sq_m` from the `_unit_conversions` dict and replace it with a line for converting `sq_ft` like so:

```diff
diff --git a/opentreemap/treemap/units.py b/opentreemap/treemap/units.py
index 2e7152b..57d1686 100644
--- a/opentreemap/treemap/units.py
+++ b/opentreemap/treemap/units.py
@@ -94,13 +94,10 @@ _unit_conversions = {
     "gal": {"gal": 1, "L": 3.785},
     "gal/year": {"gal/year": 1, "L/year": 3.785},
     "kwh/year": {"kwh/year": 1},
-    "sq_m": {"sq_m": 1, "sq_ft": 10.7639}
+    "sq_ft": {"sq_ft": 1, "sq_m": 0.092903}
 }
 _unit_conversions["ft"] = {u: v * 12 for (u, v)
                            in _unit_conversions["in"].iteritems()}
-_unit_conversions["sq_ft"] = {
-    u: v / _unit_conversions["sq_m"]["sq_ft"]
-    for u, v in _unit_conversions["sq_m"].iteritems()}
 
 
 def get_unit_name(abbrev):
```